### ISSUE TITLE
Make /saconfig clean an Automated Command

### ISSUE THREAD

**Issue by Rufure** [TotalFreedom/TotalFreedomMod#783]
Given the problems with the admin list lately, I was thinking that every Sunday on the server, it would automatically clean, and would not usable otherwise, unless triggered by the owner (in this case, Mark) or other trusted, high ranking staff.

**Comment by Wild1145** [comment]
Likely impossible until we move to a VPS...

**Comment by Wild1145** [comment]
And also would not be something we could easily build into TFM, it would probably have to be a server automated task (Like a cronjob)

**Comment by aggelosQQ** [comment]
But shouldn't we add something that will inform the active admins of the removed? Right now, we do the command and post the results on the forum so everyone will see.

**Comment by Hockeyfan360** [comment]
@Wild1145 I believe that is false. I set up an auto ban purge plugin for one of my servers that worked through java itself.

**Comment by Wild1145** [comment]
It would run well IF the server was on constantly, but I cant see any other decent way of doing it without it being a train wreck...

**Comment by Hockeyfan360** [comment]
@Wild1145 It doesn't have to be every 24 hours, it can be 12:00 central, etc.

**Comment by JeromSar** [comment]
@Wild1145 @Hockeyfan360 There are much easier ways to make this possible. For instance, we can make admin entries expire, much like bans. When an admin logs in, their entry's expiry date would be reset. Don't assume things are impossible. :)

**Comment by DUFCLiam** [comment]
This would be bad for people like me because I move to other peoples houses a lot and it would be bad because I would need to verify a lot. And I agree with @JeromSar with the expiry date.

**Comment by aggelosQQ** [comment]
@DUFCLiam I don't think you understood the point of this feature. It has nothing to do with what you are saying (unless I'm misunderstanding).

**Comment by Wild1145** [comment]
Either way just executing the command would be of little use to us. We would want some sort of log of JUST this and it to be automatically posted on the forums / somewhere that you can see who has been removed and for how long.

**Comment by DUFCLiam** [comment]
@JeromSar, Duplicate of #237

**Comment by JeromSar** [comment]
@DUFCLiam The issue might seem the same, but they're inherently different.
